libraries/c_sdk/standard/https/src/iot_https_client.c

```diff
@@ -1068,6 +1022,57 @@ static IotHttpsReturnCode_t _receiveHttpsBodySync( _httpsResponse_t * pHttpsResp
 
 /*-----------------------------------------------------------*/
 
+/**
+ * @brief Defining a network context for sending packets through the network interface.
+ * The declaration of the structure is mentioned in the transport_interface.h file.
+ */
+struct NetworkContext
+{
+    void * pNetworkConnection;                       /**< @brief The network connection used for sending packets on the network. */
+    const IotNetworkInterface_t * pNetworkInterface; /**< @brief The network interface used to send packets on the network using the above network connection. */
+};
+
+/*-----------------------------------------------------------*/
+
+static int32_t transportRecv( NetworkContext_t * pNetworkContext,
```

Comment:
I might add a comment here mentioning that this function is a dummy because the legacy library does not use it.  A reader may have missed the earlier comment.

Reply:
Sure, I will add that comment as part of a separate PR. Come to think of it, there are more doc changes that I'd like to make based on the changes here. At least this way we can start testing in CI already. Thanks Gary!